### PR TITLE
update to prismjs 1.16.0

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
refractor updated prismjs from 1.15.0 to 1.16.0. This PR keeps us in lock step to avoid duplicate dependencies.

### Additional Details
There are a couple of changes to syntax highlighting in 1.16.0 but they are minor.

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-markdown/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "github-markdown-css": "^2.4.1",
-    "prismjs": "~1.15.0",
+    "prismjs": "~1.16.0",
```
Comment (mjhenkes) [inline packages/terra-markdown/package.json]: I'm keeping the ~ for as long as we keep refractor as a dependency to avoid duplicate dependencies.

Comment (emilyrohrbough) [reply packages/terra-markdown/package.json]: What other package pulls this in? Doc-template via `react-syntax-highlighter`?

Comment (mjhenkes) [reply packages/terra-markdown/package.json]: Yes exactly

File: packages/terra-markdown/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Changelog
 Unreleased
 ----------
 ### Changed
-* Added Prismjs highlighting for code blocks
+* Added Prismjs highlighting for code blocks and ugraded to prism 1.16.0
```
Comment (StephenEsser) [inline packages/terra-markdown/CHANGELOG.md]: Small typo with `ugraded`

**Pull request merged**